json-path/src/test/java/com/jayway/jsonpath/internal/function/Issue191.java

```diff
@@ -0,0 +1,43 @@
+package com.jayway.jsonpath.internal.function;
+
+import com.jayway.jsonpath.Configuration;
+import com.jayway.jsonpath.Configurations;
+import com.jayway.jsonpath.JsonPath;
+import org.junit.Test;
+
+import java.io.InputStream;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * TDD for Issue 191
+ *
+ * Shows aggregation across fields rather than within a single entity.
+ *
+ */
+public class Issue191 extends com.jayway.jsonpath.internal.function.BaseFunctionTest {
```

Comment:
IntelliJ was having a bad day I guess, its in the same package, ugh - I'll fix this...